json_schemas/suppliers.json

```diff
@@ -33,12 +33,12 @@
       "items": {
         "type": "string"
       },
-      "minItems": 0
+      "minItems": 0,
+      "maxItems": 10
     }
   },
   "required": [
     "id",
-    "name",
-    "contactInformation"
```

Comment:
We shouldn't allow suppliers to be created without any contact information - if we have to remove it from the schema to enable supplier updates then we could add a check in the import_supplier method that there is at least one contact info?